### PR TITLE
Support Fedora's ImportExportJob

### PR DESCRIPTION
**Pull request by awead**
@projecthydra/sufia-code-reviewers
ping @escowles @ruebot

Provides support for initial import-export tool from Fedora. Not all features are available yet.

### CODE REVIEW

File: app/jobs/import_export_job.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    export(uri,
+           options.fetch(:desc_dir, CurationConcerns.config.descriptions_directory),
+           options.fetch(:bin_dir, CurationConcerns.config.binaries_directory),
+           options.fetch(:profile, nil))
+  end
+
+  private
+
+    def export(uri, desc_dir, bin_dir, _profile = nil)
+      command = "java -jar #{CurationConcerns.config.ie_jar_file_path} --mode export --resource #{uri} --descDir #{desc_dir} --binDir #{bin_dir}"
```
Comment (escowles) [inline app/jobs/import_export_job.rb]: How about `CurationConcerns.config.impexp_jar_file_path`?

**Comment by escowles** [comment]
👍

File: app/jobs/import_export_job.rb (hunk 1 above)
Comment (mjgiarlo) [inline app/jobs/import_export_job.rb]: `CurationConcerns.config.import_export_jar_file_path` pls.

File: lib/curation_concerns/configuration.rb [hunk 2]
```diff
@@ -143,6 +143,27 @@ def ingest_queue_name
       @ingest_queue_name ||= :default
     end
 
+    # @!attribute [w] ie_jar_file_path
```
Comment (mjgiarlo) [inline lib/curation_concerns/configuration.rb]: Pls expand `ie` and `impexp` to `import_export` in this file.

File: lib/generators/curation_concerns/templates/config/curation_concerns.rb [hunk 3]
```diff
@@ -53,6 +53,17 @@
   # Leaving it blank will set the start date to when ever the file was uploaded by
   # NOTE: if you have always sent analytics to GA for downloads and page views leave this commented out
   # config.analytic_start_date = DateTime.new(2014,9,10)
+
+  # Fedora import/export tool
+  #
+  # Path to the Fedora import export tool jar file
+  # config.impexp_jar_file_path = "tmp/fcrepo-import-export.jar"
```
Comment (mjgiarlo) [inline lib/generators/curation_concerns/templates/config/curation_concerns.rb]: Pls expand `impexp` to `import_export`.

**Comment by awead** [comment]
@mjgiarlo should be all fixed up now.

**Pull request merged**